src/sql-server-util/src/lib.rs

```diff
@@ -638,9 +724,6 @@ impl Connection {
                 let result = client.execute(query, &params[..]).await?;
 
                 match result.rows_affected() {
-                    [] => Err(SqlServerError::InvariantViolated(
-                        "got empty response".into(),
-                    )),
```

Comment:
I don't think anything is broken by removing this invariant but I'm open to comment. empty responses are returned for permissions changes like the DENY statements in the tests.